python/core/config_free_tier.py

```
@@ -198,87 +226,86 @@ def get_current_config() -> TierConfig:
     return True, "OK"
 
 
 if __name__ == "__main__":
     # Test configuration
     print("\n=== Configuration Test ===\n")
-    
+
     config = get_current_config()
-    
+
     print(f"\nEstimated analysis times:")
```

Ruff (0.14.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)